sdk/include/aditof/camera.h

```diff
@@ -123,6 +123,7 @@ class SDK_API Camera {
      * @return Status
      */
     virtual Status requestFrame(Frame *frame,
+                                aditof::BufferInfo *bufferInfo = nullptr,
```

Comment:
BufferInfo is a low level API structure. Doesn't make sense to show up in the high level API.

Reply:
To get the timestamp into the Frame object, we should create a timestamp field for it and copy the value from the BufferInfo object. And this should happen in the camera implementation.

Reply:
Removed bufferInfo from high level API